### PR TITLE
Unit cardinality restrictions closes #69

### LINKED ISSUE #69: Should we have multiple numerators and denominators for RatioUnits?
In a prior internal (not really a) release of gist9.0.0 and other prior versions, RatioUnit has exactly 1 UnitOfMeasure as a numerator and exactly one as a denominator. Now it can have more than one. 
I cannot work out what that means.  How might that ever be useful?

### PR DESCRIPTION
**Pull request by daliadahleh**
- Added cardinality restrictions to AreaUnit and VolumeUnit
- Made unit classes disjoint

### CODE REVIEW

**Comment by uscholdm** [comment]
See my comments on the issue #69 which I made before I saw the PR.

File: OntologyFiles/gistTop.owl [hunk 1]
```diff
@@ -658,15 +670,15 @@
...
-		<owl:disjointWith rdf:resource="&gist;Template"/>
+		<owl:disjointWith rdf:resource="&gist;Content"/>	
+		<owl:disjointWith rdf:resource="&gist;GeoPoint"/>	
+		<owl:disjointWith rdf:resource="&gist;GeoRegion"/>	
+		<owl:disjointWith rdf:resource="&gist;IntellectualProperty"/>	
+		<owl:disjointWith rdf:resource="&gist;Language"/>	
+		<owl:disjointWith rdf:resource="&gist;Organization"/>	
+		<owl:disjointWith rdf:resource="&gist;PhysicalIdentifiableItem"/>	
+		<owl:disjointWith rdf:resource="&gist;SchemaMetaData"/>	
+		<owl:disjointWith rdf:resource="&gist;Template"/>
```
[the remark this answers is not in the document]
Comment (daliadahleh) [reply OntologyFiles/gistTop.owl]: Done

File: OntologyFiles/gistTop.owl [hunk 2]
```diff
@@ -697,11 +709,13 @@
 					</rdf:Description>
 					<owl:Restriction>
 						<owl:onProperty rdf:resource="&gist;multiplicand"/>
-						<owl:someValuesFrom rdf:resource="&gist;DistanceUnit"/>
+						<owl:onClass rdf:resource="&gist;DistanceUnit"/>
```
[the remark this answers is not in the document]
Comment (daliadahleh) [reply OntologyFiles/gistTop.owl]: Agreed, changed to union of both.

**Comment by rjyounes** [comment]
Reassigning to @sa-bpelakh to do the merge.

**Comment by sa-bpelakh** [comment]
@daliadahleh There is one more issue, that @uscholdm correctly raised - in the files you edited, the XMLSchema namespace was renamed from `xs:` to `xsd:`, which we mean to address separately under https://github.com/semanticarts/gist/issues/158. If you cal please roll back that change, I think we'll be good to go. Thanks!

**Comment by uscholdm** [comment]
The chagnes I requested have been done.  Is there a way to acknowledge this fact to git w/o approving the PR? I raised another question, but it was not a change request.

**Comment by daliadahleh** [comment]
> @daliadahleh There is one more issue, that @uscholdm correctly raised - in the files you edited, the XMLSchema namespace was renamed from `xs:` to `xsd:`, which we mean to address separately under #158. If you cal please roll back that change, I think we'll be good to go. Thanks!

I reverted the change in gistUnit.owl. It seems that gistTop.owl already had xsd so I kept it as is.

**Comment by rjyounes** [comment]
Peter: Add disjoints in a separate OWL file, can be added to some application profiles and not others. 
Rebecca: When would we use OWL application profiles vs SHACL?
Dan: use case of Persons not being Ownables - if not in gist, then can be in a distinct OWL application profile.

Michael: Proposes adding/keeping disjoints between base units and remove any that exist between the product and ratio units. 

Dave: The reason we have a lot of disjoints is that otherwise you can't use the inference engine to find inconsistencies in the ontology. We should add them where they are useful. We like to do disjoints as high up in the tree as possible. Most problems are avoided that way. Make base units and magnitudes disjoint. Make product and ratio disjoint from the base units, which will take care of all the lower ones with product and ratio units.
Michael: then can't determine that Area is different from Volume, since neither are base units. If two units are disjoint, then logically the magnitudes based on those units are disjoint, but can't express that in OWL, need SHACL.

Boris: if base units are disjoint from one another, you can load all units and infer all the specific disjoints. Error-prone to try to keep track of all explicit disjoints. 

Dave: separate meaning from usage. Having optional disjoints (as in Peter's application profile suggestion) allows different usages to have different meanings for the same terms, which we don't want. When people specialize gist, they can add disjoints and other axioms.
Peter: this is what I was proposing.

Dalia: All these definitions are in gistTop rather than gistUnit.
Dave: GeoRegion needs units, and GeoRegion is in gistTop, so units had to be in gistTop. Suspect we eventually we de-modularize.

Consensus: Only base units will be defined as disjoint.

**Pull request merged**